src/python/pants/notes/2.0.x.rst

```diff
@@ -5,6 +5,231 @@ This document describes releases leading up to the ``2.0.x`` ``stable`` series.
 
 See https://www.pantsbuild.org/v2.0/docs/release-notes-2-0 for an overview of the changes in this release.
 
+2.0.0.dev7 (8/10/2020)
+----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* A python_distribution target. (#10565)
+  `PR #10565 <https://github.com/pantsbuild/pants/pull/10565>`_
+
+* Enable Python dependency inference by default (#10520)
+  `PR #10520 <https://github.com/pantsbuild/pants/pull/10520>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Allow using file addresses on the command line (#10573)
+  `PR #10573 <https://github.com/pantsbuild/pants/pull/10573>`_
+
+* Targets act as aliases for their files (#10511)
+  `PR #10511 <https://github.com/pantsbuild/pants/pull/10511>`_
+
+* Add `--build-patterns` option (#10559)
+  `PR #10559 <https://github.com/pantsbuild/pants/pull/10559>`_
+
+* Allow protobuf_library targets to specify a python source root. (#10549)
+  `PR #10549 <https://github.com/pantsbuild/pants/pull/10549>`_
+
+* Revert "Add cache_failures option to Process type (#10433)" (#10534)
+  `PR #10534 <https://github.com/pantsbuild/pants/pull/10534>`_
```

Comment:
Because this was reverted, you can put it in "Refactoring" for a cleaner changelog. This isn't really a new feature.

Reply:
It's a revert of something else in this section, so it made sense to me to pair them up. I can put both in refactoring, but that's kinda weird. If we had done a release before reverting, each of these would have been in this section.

Reply:
Indeed, I think they should both be moved. My reasoning is that they're both in the same release, so why bother making the user see this happened? It has no end impact on them that we added this feature then reverted it before being consumable.

Reply:
I'll move them both though.